packages/e2e-test-utils/src/get-edited-post-content.js

```diff
@@ -9,5 +9,17 @@ import { wpDataSelect } from './wp-data-select';
  * @return {Promise} Promise resolving with post content markup.
  */
 export async function getEditedPostContent() {
+	const content = page.evaluate( () => {
+		if ( ! window._getContent ) {
+			return;
+		}
+
+		return window._getContent();
```

Comment:
Not sure how this should best be done. Alternatively we expose everything though `window.wp` as well, or we create a text field with the value.

Reply:
I feel we shouldn't try to touch the `e2e-test-utils` or at least we should have specific WordPress utils, Playground utils and potentially Common utils too. These can still live in this utils package but it definitely need more organization. Right now it's just something where we put everything without thinking of organizing it properly especially because most of these utils have "context" in which they can be used in or not.

Reply:
I agree. This was a temporary thing. I'm not sure what's the best way to check content for the playground right now as well.